### ISSUE TITLE
No available dynamic library for centos6.x86_64 with features None

### ISSUE THREAD

**Issue by newbieh4cker** [dashingsoft/pyarmor#312]
![image](https://user-images.githubusercontent.com/23703741/90469545-10962800-e154-11ea-9962-8c35d998d76c.png)

**Comment by newbieh4cker** [comment]
used command  `pyarmor pack -x " --platform centos6.x86_64" test.py`

**Comment by jondy** [comment]
As a quick workaround, apply this patch in pyarmor package source file `platforms/index.json`:
```patch
@@ -216,7 +216,7 @@
         },
         {
             "id": "centos6.x86_64.7",
-            "name": "centos.x86_64",
+            "name": "centos6.x86_64",
             "platform": "linux",
             "path": "centos6.x86_64",
             "machines": ["x86_64", "amd64", "x64"],
```

**Comment by jondy** [comment]
Fixed in v6.4.0
